src/client/eppo-client.ts

```diff
@@ -124,7 +126,9 @@ export interface IEppoClient {
     defaultValue: object,
   ): object;
 
-  setLogger(logger: IAssignmentLogger): void;
+  setAssignmentLogger(assignmentLogger: IAssignmentLogger): void;
```

Comment:
this will be a breaking change due to API rename. maybe consider deprecating the old API  first and delegating to the new method

Reply:
EXCELLENT catch!